src/Ryujinx.Ava/UI/Models/TitleUpdateModel.cs

```diff
@@ -8,7 +8,12 @@ public class TitleUpdateModel
         public ApplicationControlProperty Control { get; }
         public string Path { get; }
 
-        public string Label => LocaleManager.Instance.UpdateAndGetDynamicValue(LocaleKeys.TitleUpdateVersionLabel, Control.DisplayVersionString.ToString());
+        private bool IsXCI => System.IO.Path.GetExtension(Path)?.ToLower() == ".xci";
```

Comment:
```suggestion
        private bool IsXci => Path.GetExtension(Path)?.ToLower() == ".xci";
```
And a using to `System.IO` ?

Reply:
It would use the Path defined above.